OWNERS

```diff
@@ -1,3 +1,3 @@
 approvers:
 - openshift-storage-maintainers
-component: "Storage"
\ No newline at end of file
+component: "Storage / Kubernetes External Components"
```

Comment:
In Azure Disk CSI Driver we use a `subcomponent` field: https://github.com/openshift/azure-disk-csi-driver/blob/master/OWNERS

Which way is the correct one?

Reply:
This one. I have asked about it. :) We no longer have subcomponents in our bug tracking tools...